en/how-to/create-site-to-site-vpn-on-vpc.md

```diff
@@ -1,72 +1,43 @@
-
-
-#### CloudMonkey commands:
-
-- `create vpngateway vpcid=<VPC_ID>`
-- `create vpncustomergateway dpd=true esppolicy=aes128-sha1;modp1536 forceencap=false ikepolicy=aes128-sha1;modp1536 ipsecpsk=<Unique +10 alphanumeric> cidrlist=<CIDR> name=<NAME> projectid=<PROJECT_ID> gateway=<REMOTE_PUBLIC_IP>`
-- `create vpnconnection s2scustomergatewayid=<PREVIOUS_COMMAND_OUTPUT_UUID> s2svpngatewayid=<FIRST_COMMAND_OUTPUT_UUID>`
-
-
-##### Example:
-- `create vpngateway vpcid=d86cabde-bd61-4884-8166-56fdd2fdf66e`
-- `create vpncustomergateway dpd=true esppolicy=aes128-sha1;modp1536 forceencap=false ikepolicy=aes128-sha1;modp1536 ipsecpsk=StrongSecret cidrlist=10.178.52.0/22 name=vpc_to_office projectid=db3ed20c-fc05-4072-b0a3-608e33feb7b0 gateway=69.196.164.512`
-- `create vpnconnection s2scustomergatewayid=7542cca9-8a6d-4bb6-811f-e08826df3955 s2svpngatewayid=30c3ecf9-7f76-4fea-abd7-cb16d4870502`
-
-
-At this point, the configuration must be configured on the Office or remote VPN side where it will use the VPC SourceNAT IP as the remote site IP. then the connectivity between the two network should be working.
+A site-to-site VPN is useful for interconnecting a VPC to a remote office, another data centre, or to another VPC.
+
+The following example illustrates how to connect two VPCs with a site-to-site VPN.  Details will vary for your particular devices.
+
+#### Example VPC details
+| VPC name | Environment name | VPC CIDR | Source NAT IP |
+| --- | --- | --- | --- |
+| dmz | dmz-env | 10.0.0.0/22 | 172.30.200.106 |
+| management | mgt-env| 10.0.4.0/22 | 172.30.200.107 |
+
+#### Site-to-site VPN configuration
+1. Go to **dmz-env** environment.
+1. Go to *Networking* tab.
+1. Click on the gear menu for *Site-to-site VPNs*.
+1. Click on *Add site-to-site VPN* button in upper right of page.
+1. Enter the details for the tunnel from **dmz** to **management**:
+   - **Name:** Tunnel to management
+   - **Remote public IP:** 172.30.200.107
+   - **Remote CIDR:** 10.0.4.0/22
+   - **IPSec pre-shared key:** Uj2nzrTQ7xkbgun3ZqVFPbyxr9wfQzXZG5ZJ
+   - **Dead Peer Detection:** Enabled
+   - **Force encapsulation:** Disabled
+   - **Passive connection:** Enabled
+   - Accept defaults for all other tunnel options.
+1. Click *Submit*
+1. Go to **mgt-env** environment and repeat steps 2 though 4.
+1. Enter the details for the tunnel from **management** to **dmz**:
+   - **Name:** Tunnel to dmz
+   - **Remote public IP:** 172.30.200.106
+   - **Remote CIDR:** 10.0.0.0/22
+   - **IPSec pre-shared key:** Uj2nzrTQ7xkbgun3ZqVFPbyxr9wfQzXZG5ZJ
+   - **Dead peer detection:** Enabled
+   - **Force encapsulation:** Disabled
+   - **Passive connection:** Disabled
+   - Accept defaults for all other tunnel options
+1. Click *Submit*
+1. The list of Site-to-site VPNs now lists **Tunnel to dmz**.  The status will say **Connected**.
```

Comment:
Should we state the same for Tunnel to Management?

Reply:
You know, I had originally thought I wouldn't, but since you mentioned it, it feels incomplete, so I will add some text about the other tunnel.

Reply:
Ok, I added a sentence to the English and French.  What do you think?